### ISSUE TITLE
Missing export default in file-saver dependency

### ISSUE THREAD

**Issue by katafractari** [egeriis/zipcelx#54]
I'm getting
```
 WARN  in ./node_modules/zipcelx/lib/module.js
"export 'default' (imported as 'FileSaver') was not found in 'file-saver'
```
when using it in a Nuxt.js project. My import statement looks like this:

`import zipcelx from 'zipcelx'`

file-saver dependency resolves to 1.3.8, which does not do a ES6 style "export default".
I'm using zipcelx 1.5.0.

**Comment by katafractari** [comment]
Workaround for this issue is to install the latest file-saver dependency.

`npm i -S file-saver@2.0.0`

**Comment by egeriis** [comment]
this may be fixed with #56
